projects/plugins/search/composer.json

```diff
@@ -41,6 +41,8 @@
 	"prefer-stable": true,
 	"extra": {
 		"mirror-repo": "Automattic/jetpack-search",
+		"release-branch-prefix": "search",
+		"wp-plugin-slug": "jetpack-search",
```

Comment:
I didn't add the plugin slug since we don't know it yet (though safe to assume). I'll open an issue about adding the release-branch-prefix.

Reply:
https://github.com/Automattic/jetpack/pull/20819